docs/articles/documentation/test-api/actions/debug.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+title: Debug
+permalink: /documentation/test-api/actions/debug.html
+checked: true
+---
+# Debug
+
+Pauses the test and allows you to debug using the browser's developer tools.
+
+```text
+t.debug()
+```
+
+When test execution reaches this action, it pauses so that you can open browser's developer tools
+and check the web page state, the location of DOM elements, their CSS styles, etc.
+
+The footer displays buttons that allow you to continue test execution or step to the next test action.
+
+![Page Footer when Debugging on a Client](../../../images/debugging/client-debugging-footer.png)
```

Comment:
It would be nice to say also, that TestCafe logs where the test is stopped in the console: https://github.com/DevExpress/testcafe/pull/1116#issuecomment-274753029